app/javascript/components/student/MentoringSession.tsx

```diff
@@ -50,13 +51,14 @@ export const MentoringSession = ({
   links,
 }: {
   userId: number
-  discussion?: MentorDiscussion
+  discussion?: Discussion
+  mentor?: Mentor
```

Comment:
I brought `mentor` to be next to `discussion` instead of inside it, so that we can use the same `MentorSessionDiscussion` type for both contexts. The other already had the `student` type next to it.

Reply:
Sounds good.